cockroachcloud/cockroachdb-dedicated-on-azure.md

```diff
@@ -0,0 +1,135 @@
+
+### Can we export logs and metrics from a dedicated cluster on Azure to Azure Monitor or a third-party observability service?
+
+During limited access, exporting metrics to Datadog is supported. Refer to [Export Metrics From a {{ site.data.products.dedicated }} Cluster](export-metrics.html). It’s not possible to export cluster logs or metrics to Azure Monitor or to another third-party observability service during the limited access period.To express interest in this feature, contact your Cockroach Labs account team.
+
+### Are CockroachDB user-defined functions available for dedicated clusters in Azure?
+
+Yes, [user-defined functions](/docs/{{site.versions["stable"]}}/user-defined-functions.html) are supported for {{ site.data.products.dedicated }} clusters on Azure. The same CockroachDB binaries is used across {{ site.data.products.db }} deployment environments, and all SQL features behave the same on Azure as on GCP or AWS, with the exception of multi-region capabilities during the limited access period.
+
+### Can we use {{ site.data.products.dedicated }} on Azure if we are coming from PostgreSQL?
+
+CockroachDB supports the [PostgreSQL wire protocol](https://www.postgresql.org/docs/current/protocol.html) and the majority of PostgreSQL syntax. Refer to [Supported SQL Feature Support](/docs/{{site.versions["stable"]}}//sql-feature-support.html). The same CockroachDB binaries is used across {{ site.data.products.db }} deployment environments, and all SQL features behave the same on Azure as on GCP or AWS, with the exception of multi-region capabilities during the limited access period.
+
+### How are {{ site.data.products.dedicated }} clusters on Azure isolated from each other? Do they follow a similar approach like on AWS and GCP?
+
+We follow a similar tenant isolation approach on Azure as on GCP and AWS. During the limited access period, each {{ site.data.products.dedicated }} cluster is created its own unique Azure subscription on a [AKS cluster](https://azure.microsoft.com/en-us/products/kubernetes-service) in a unique [VNet](https://learn.microsoft.com/en-us/azure/virtual-network/virtual-networks-overview). Implementation details are subject to change.
+
+### Can we use Single-Sign On to sign-in to {{ site.data.products.db }} to manage {{ site.data.products.dedicated }} clusters on Azure?
+
+Yes, [Cloud Organization SSO](cloud-org-sso.html) is supported. This feature is unrelated to the cluster's deployment environment.
+
+### Is it possible to use a secure and centralized authentication method for {{ site.data.products.dedicated }} clusters on Azure?
```

Comment:
Client certs is supported for both human and app users - https://www.cockroachlabs.com/docs/cockroachcloud/client-certs-dedicated.html